JBang ERROR: Script or alias could not be found or read: '.jbang.version=3.21.0'

When running a YAML from VSCode for Windows, the following error occurs:
`[jbang] [ERROR] Script or alias could not be found or read: '.jbang.version=3.21.0'`

This is due to Karavan generating the jbang command without wrapping the JVM parameters in quotes, that is required under Powershell.

This is the command generated:
`jbang -Dcamel.jbang.version=3.21.0 camel@apache/camel run *`

This is what it should have been:
`jbang "-Dcamel.jbang.version=3.21.0" camel@apache/camel run *`

Follows screenshot:
![image](https://github.com/apache/camel-karavan/assets/1853562/517f028d-4c42-46a5-9d45-f0c92db8d6fd)

--- a/karavan-vscode/src/jbang.ts
+++ b/karavan-vscode/src/jbang.ts
@@ -46,7 +46,7 @@ export async function camelJbangGenerate(rootPath: string, openApiFullPath: stri
 
 function prepareCommand(command: string): string {
     const version = workspace.getConfiguration().get("camel.version");
-    return "jbang -Dcamel.jbang.version=" + version + " camel@apache/camel " + command;
+    return "jbang \"-Dcamel.jbang.version=" + version + "\" camel@apache/camel " + command;
 }
 
 export function camelJbangRun() {

